plugins/vendor/zoom/lib/model/oAuth/kZoomOauth.php

```diff
@@ -108,6 +109,25 @@ protected static function getHeaderData()
 		return array($zoomBaseURL, $redirectUrl, $header, $userPwd);
 	}
 
+	protected static function getHeaderDataByPartnerId($partnerId)
```

Comment:
to support multiple accounts lets change sections to [zoomAccount_accountid_partnerid] while  [zoomAccount] is the fallback
 specifically on getZoomUrl as well

Reply:
done